src/components/App.js

```diff
@@ -2,6 +2,15 @@ import React, { Suspense, lazy, useRef, useState, useEffect, useCallback } from
 import { useDispatch, useSelector } from "react-redux";
 import { Redirect, Route, Switch, useHistory } from "react-router-dom";
 import { ChartConfigProvider } from "bento-charts";
+import {
+    fetchOpenIdConfiguration,
+    createAuthURL,
+    useHandleCallback,
+    getIsAuthenticated,
+    refreshTokens,
+    tokenHandoff,
+    LS_SIGN_IN_POPUP,
```

Comment:
this was moved out, but the actual callbacks related to popup handling were not - might be nice to consolidate in the future

Reply:
WIP: https://github.com/bento-platform/bento_auth_js/pull/2